server/storage_index.go

```diff
@@ -483,6 +485,28 @@ func (si *LocalStorageIndex) mapIndexStorageFields(userID, collection, key, vers
 	rv.AddField(bluge.NewNumericField("read", float64(read)).StoreValue())
 	rv.AddField(bluge.NewNumericField("write", float64(write)).StoreValue())
 
+	if len(sortFilters) > 0 {
```

Comment:
This is duplicating the indexing of values that happens in `BlugeWalkDocument`. You could possibly set the `Sortable` property directly there, but keep in mind that other features also depend on this function.

Reply:
after BlugeWalkDocument set the Sortable property by sortFilters,and the sortFilters must start with "value.". I didn't modify BlugeWalkDocument as other features also depend on this function.

Reply:
@ftkg

Reply:
Thanks @hexun80149128 we'll evaluate how best to arrange this part of the code before merging it.

Reply:
@hexun80149128 can you please sign up the CLA so we can accept this PR later?

Reply:
@ftkg done.